.gitignore

```diff
@@ -116,4 +116,7 @@ build/
 /lastRestart.py
 /.factorypath
 start.yml
+config
+src/main/resources/resource/InMoov2
```

Comment:
why are these (ProgramAB / InMoov2 resources ) git ignored?

Reply:
Because during development, they are the most convenient place to clone them, without going into the complexities of using git submodules